src/modules/menu/components/menu/MenuTray.vue

```diff
@@ -1,10 +1,102 @@
+<script setup>
+import { computed, onMounted, ref, toRefs, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import MenuThreeD from '@/modules/menu/components/3d/MenuThreeD.vue'
+import MenuActiveLayersList from '@/modules/menu/components/activeLayers/MenuActiveLayersList.vue'
+import MenuAdvancedToolsList from '@/modules/menu/components/advancedTools/MenuAdvancedToolsList.vue'
+import MenuSection from '@/modules/menu/components/menu/MenuSection.vue'
+import MenuPrintSection from '@/modules/menu/components/print/MenuPrintSection.vue'
+import MenuSettings from '@/modules/menu/components/settings/MenuSettings.vue'
+import MenuShareSection from '@/modules/menu/components/share/MenuShareSection.vue'
+import MenuTopicSection from '@/modules/menu/components/topics/MenuTopicSection.vue'
+
+const dispatcher = { dispatcher: 'MenuTray.vue' }
+
+const i18n = useI18n()
+const store = useStore()
+
+const props = defineProps({
+    compact: {
+        type: Boolean,
+        default: false,
+    },
+})
+const { compact } = toRefs(props)
+
+const refs = ref({})
+// multiMenuSections means that they can be open together
+const multiMenuSections = ref(['topicsSection', 'activeLayersSection'])
+// singleModeSections means that those section cannot be open together with other
+// sections and would therefore toggle other sections automatically.
+const singleModeSections = ref([
+    'drawSection',
+    'settingsSection',
+    'shareSection',
+    'toolsSection',
+    'printSection',
+    '3dSection',
```

Comment:
I think we should let this be in the `multiMenuSections`, I can see users wanting to de-activate vegetation/building while setting up or ordering up layers, and be bothered that "Map displayed" has been closed when changing 3D settings